hail/src/main/scala/is/hail/linalg/GridPartitioner.scala

```diff
@@ -117,14 +118,15 @@ case class GridPartitioner(blockSize: Int, nRows: Long, nCols: Long, maybeBlocks
   def transpose: (GridPartitioner, Int => Int) = {
     val gpT = GridPartitioner(blockSize, nCols, nRows)
     def transposeBI(bi: Int): Int = gpT.coordinatesBlock(this.blockBlockCol(bi), this.blockBlockRow(bi))
-    def inverseTransposeBI(bi: Int) = this.coordinatesBlock(gpT.blockBlockCol(bi), gpT.blockBlockRow(bi))
     maybeBlocks match {
       case Some(bis) =>
-        val (biTranspose, piTranspose) = bis.map(transposeBI).zipWithIndex.sortBy(_._1).unzip
-        val inverseTransposePI = piTranspose.zipWithIndex.sortBy(_._1).map(_._2)
+        val (biTranspose, newPIToOldPI) = bis.map(transposeBI).zipWithIndex.sortBy(_._1).unzip
```

Comment:
this is `transposedBIToUntranposedBI`, right? `newPIToOldPI` has nothing to do with partition indices.

Reply:
Are you commenting on the first or second of the tuple elements. `biTranspose` is the result of mapping the `blockIndices` from the old `GridPartitioner` to the new transposed one. `newPIToOldPI` is the mapping from new partition indices to old partition indices. Maybe better to call it something about "parent" partition indices?